inc/em_configuration.h

```diff
@@ -102,8 +102,9 @@ class em_configuration_t {
     virtual void set_primary_device_type(char *) = 0;
     virtual em_freq_band_t get_band() = 0;
     virtual void set_band(em_freq_band_t band) = 0;
-
     virtual em_network_ssid_info_t *get_network_ssid_info_by_haul_type(em_haul_type_t haul_type) = 0;
+    virtual em_rd_freq_band_t map_freq_band_to_rf_band(em_freq_band_t band) = 0;
+    em_freq_band_t convert_freq_band(em_freq_band_t band);
```

Comment:
input value and return value seems to be same enum. Is this correct?

Reply:
Yes.